### DISCUSSION TITLE
IconLayer Icons Appear Larger In 8.5.x vs 8.4.12?

### DISCUSSION THREAD

**Discussion by jfuehner** [visgl/deck.gl#6065] [Q&A] [answered]
I noticed when I upgraded to v8.5.x icons are appear larger than they did in v8.4.12...

CodePen using latest deck; v8.5.x
https://codepen.io/jfuehner/pen/ZEKMgOV

CodePen using deck v8.4.12
https://codepen.io/jfuehner/pen/mdmGNRR

Is this a known issue?

**Comment by Pessimistress** [comment, marked as answer]
> is this a known issue?

If you search for "IconLayer" in issues...

https://github.com/visgl/deck.gl/issues/6039#issuecomment-889151173

**Reply by jfuehner** [reply]
@Pessimistress for `PathLayer` a guide to fallback to the old size was provided.

> PathLayer with billboard: true After upgrading to v8.5, in order to maintain the same appearance, you need to multiply 2/3 to the objects' width/size. This can be done by either changing the accessor (getWidth/getSize) or the scaling prop (sizeScale/widthScale).

Would there be something similar you might recommend for `IconLayer `and `TextLayer` to achieve a similar appearance to 8.4.x?

**Reply by Pessimistress** [reply]
`sizeScale` is an IconLayer/TextLayer prop.
